frontend/src/components/Team/TeamCard.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <div class="row">
+    <div class="col-sm" v-for="teamMate in teamMate" :key="teamMate.id">
```

Comment:
```suggestion
    <div class="col-sm" v-for="teamMate in teamMates" :key="teamMate.id">
```
It is not very useful here, but in more complex components, it allows for less of a headache to name arrays with a plural noun.

Reply:
I am also not sure of the `col-sm` class. What were you intending? You could use `col-sm-x` where `x` is the number of columns you intend to use. `sm` is used for media queries to have a property applied from extra small devices and up, but you do not give any number of columns.
Also note (though I am not entirely sure), that `<div class="col" ...>` is equivalent to `<v-col ...>`. In any case, use vuetify components the more you can.